integration/operator_test.go

```diff
@@ -98,8 +102,24 @@ func TestMain(m *testing.M) {
 		os.Exit(1)
 	}
 
+	if os.Getenv("VERSION_BUNDLE_VERSION") == "" {
+		version := f.GetVersionBundleVersion(service.NewVersionBundles(), os.Getenv("TESTED_VERSION"))
+		if version == "" {
+			log.Printf("No version bundle version for TESTED_VERSION %q", os.Getenv("TESTED_VERSION"))
+			os.Exit(0)
+		}
+		os.Setenv("VERSION_BUNDLE_VERSION", version)
+	}
+	log.Printf("TESTED_VERSION %v", os.Getenv("TESTED_VERSION"))
+	log.Printf("VERSION_BUNDLE_VERSION %v", os.Getenv("VERSION_BUNDLE_VERSION"))
+
 	c = newAWSClient()
 
+	if err := createHostPeerVPC(); err != nil {
+		log.Printf("unexpected error: %v\n", err)
+		os.Exit(1)
+	}
```

Comment:
Is this change related? Can this be split into another PR?

Reply:
Yes, it is required for the parallel execution, the PR involves getting WIP and current versions and running them, do you think it is better to split?

Reply:
I would appreciate that.

Reply:
ok done in #729, once that one lands I'll merge master here, the changes there are required here.

Reply:
👍 I reviewed that one already. Ping me when this one is ready to review.

Reply:
@kopiczko ready for review, now only changes related to parallel execution here.